test/jdk/java/util/concurrent/locks/Lock/OOMEInAQS.java

```diff
@@ -0,0 +1,107 @@
+//package concurrent;
```

Comment:
Do you mind adding a copyright header to the test?

Reply:
OK, done.